plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -105,7 +112,16 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
         else -> unimplementedTypeEmbedding(type)
     }
 
-    override fun getField(field: FirPropertySymbol): FieldEmbedding? = fields[field.callableId.embedClassMemberName()]
+    /**
+     * Retrieves the corresponding field embedding for a property with a **backing field**,
+     * using its property symbol. If the passed `field` represents an extension property,
+     * the function returns null.
+     */
```

Comment:
Again, try to not restate what's already clear from the code.  The key here is that if the property has a backing field, get it.  Whether it's an extension property or not is not so important, I think?